src/App.css

```diff
@@ -39,52 +68,199 @@
 
 .bookContainer {
   display: flex;
+  flex-direction: row;
   justify-content: space-between;
-  padding: 4rem;
-  border: 1px solid black;
-  gap: 1rem;
-  align-items: center;
+  background-color: #fff;
+  margin: 1% 5%;
+  padding: 1% 10% 1% 2%;
+  font-family: 'Roboto Slab', serif;
+  border: solid 1px #e8e8e8;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
 }
 
-.buttonsList {
+.bookTitle {
   display: flex;
+  flex-direction: column;
   gap: 1rem;
 }
 
-.bookTitle {
+.category {
+  color: #121212;
+  opacity: 0.5;
+  font-weight: bold;
+}
+
+.title {
+  color: #121212;
+  font-size: larger;
+  font-weight: bold;
+}
+
+.author {
+  color: #4386bf;
+}
+
+.buttonsList {
   display: flex;
-  flex-direction: column;
-  gap: 0.5rem;
-  width: 20rem;
+  gap: 1rem;
 }
 
 .buttonsList li {
   list-style: none;
 }
 
-.bookChart {
+.bookBtn {
+  outline: none;
+  border: none;
+  background-color: #fff;
+  color: #4386bf;
+  cursor: pointer;
+  padding-left: 0%;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+}
+
+.progress {
   display: flex;
-  gap: 0.5rem;
-  width: 10rem;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  min-width: 35%;
 }
 
-.bookChapter {
+.chart {
   display: flex;
-  flex-direction: column;
-  gap: 0.5rem;
-  width: 20rem;
+  flex-direction: row;
+  align-items: center;
 }
 
-.addBook {
-  margin-top: 1rem;
+.ProgressBar {
+  margin-right: 10%;
+}
+
+.circle-wrap {
+  width: 75px;
+  height: 75px;
+  background: #e8e8e8;
+  border-radius: 50%;
+  border: 1px solid #cdcbd0;
+}
+
+.mask .fill {
+  clip: rect(0, 37.5px, 75px, 0);
+  background-color: #227ded;
+}
+
+.mask.full,
+.circle .fill {
+  animation: fill ease-in-out 3s;
+  transform: rotate(135deg);
+}
+
+.circle-wrap .circle .mask,
+.circle-wrap .circle .fill {
+  width: 75px;
+  height: 75px;
+  position: absolute;
+  border-radius: 50%;
+}
+
+.circle-wrap .circle .mask {
+  clip: rect(0, 75px, 75px, 37.5px);
+}
```

Comment:
- [ ] I couldn't see the circle that describe the progress in your design, please 🙏🏽 add it to be similar to the Zeplin design,